include/rocksdb/trace_record.h

```diff
@@ -0,0 +1,133 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+// To do: ROCKSDB_LITE ?
+
+#include "rocksdb/db.h"
+#include "rocksdb/slice.h"
+#include "rocksdb/write_batch.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+// Supported Trace types.
+enum TraceType : char {
+  kTraceNone = 0,
+  kTraceBegin = 1,
+  kTraceEnd = 2,
+  kTraceWrite = 3,
```

Comment:
Between L20 and L21, add comment "Query level tracing related trace types" and replace L33. L25 changes to "Block cache tracing related trace types" and L31 changes to "IO tracing related trace types"

Reply:
ack